human_tests/Human_latent_transform_simulation.ipynb

```diff
@@ -315,17 +303,15 @@
     "simulated_data = simulate_expression_data.simulate_by_latent_transformation(\n",
     "    num_simulated_experiments,\n",
     "    normalized_data_filename,\n",
-    "    NN_architecture,\n",
```

Comment:
oh interesting -- `NN_architecture` was used previously to make the directory name, but now since its user-specified, this is no longer necessary.

Reply:
Yup exactly. At the time I had my folders organized in a certain way and it didn't occur to me that there would be other directory organizations:) I'm glad that things are more flexible now